### DISCUSSION TITLE
SSR mode Error: Hydration completed but contains mismatches.

### DISCUSSION THREAD

**Discussion by jennyluo0607** [quasarframework/quasar#10805] [CLI - SSR mode] [answered]
Run the code and report this error. I found that the problem is on the following QTabPanel. How can I solve it?
![Snipaste_2021-09-26_10-43-17](https://user-images.githubusercontent.com/87688455/134791478-f8fda86a-726f-4d72-a73d-f97709508873.jpg)

**Comment by ontwikkelfabriek** [comment, marked as answer]
@jennyluo0607 

>  I have searched it on other websites because the client and server do not synchronize the store

I googled other reasons why this error can occur:

 **When does the problem occur?**
_If a component is initially mounted or hydrated in the DOM, it needs to precisely correspond to the HTML generated on the server side. Only after this has happened may the HTML change. A suitable point in time for this after hydration would therefore be the mounted lifecycle._

See:
https://www.sumcumo.com/en/understand-and-solve-hydration-errors-in-vue-js

**Reply by jennyluo0607** [reply]
Thank you very much！

**Comment by ontwikkelfabriek** [comment]
@jennyluo0607 

Please keep us posted if and how you fixed your problem!
